iroh-net/src/hp/derp/clients.rs

```diff
@@ -63,19 +65,45 @@ impl Client {
     }
 
     pub fn send_packet(&self, packet: Packet) -> Result<(), SendError> {
-        try_send(&self.conn.client_channels.send_queue, packet)
+        let res = try_send(&self.conn.client_channels.send_queue, packet);
+        if res.is_ok() {
+            inc!(Metrics, send_packets_sent);
+        }
```

Comment:
why not mark as `dropped`

Reply:
I'll leave a comment. If we cannot send the packet here, it's still possible that we are able to forward it.